addons/spreadsheet/static/src/global_filters/components/records_selector.js

```diff
@@ -93,17 +91,17 @@ X2ManyTagSelector.template = xml/*xml*/ `
             </div>
         </div>
     </div>`;
-X2ManyTagSelector.props = {
+RecordsSelector.props = {
     /**
      * Callback called when a record is selected or removed.
      * (selectedRecords: Array<{ id: number; display_name: string }>) => void
      **/
     onValueChanged: Function,
-    relatedModel: String,
+    resModel: String,
     /**
      * Array of selected record ids
      */
-    selectedValues: {
+    resIds: {
```

Comment:
other idea ? any thoughts ? `activeIds` ? `selectedIds` ?

Reply:
resIds is perfect IMO